### PR TITLE
Feature#44/integration movie details with wrapper

### PR DESCRIPTION
**Pull request by q270311**

### CODE REVIEW

File: src/features/movieDetails/index.js [hunk 1]
```diff
@@ -1,77 +1,100 @@
...
+                                    }
+                                    alt="backdrop"
+                                />
+                            </PosterWrapper>
+                            <BackdropVotes
+                                voteAverage={details.vote_average}
+                                voteCount={details.vote_count}
+                            />
+                        </BackdropWrapper>
+                    }
```
Comment (kepkaklaudia) [inline src/features/movieDetails/index.js:53]: When we don't have a Backdrop Poster, we don't want to display the entire section, not just the poster

File: src/features/movieDetails/index.js [hunk 2]
```diff
@@ -1,77 +1,100 @@
...
+                        <BackdropWrapper>
+                            <Title>
+                                {details.original_title}
+                            </Title>
+                            <PosterWrapper>
+                                <Backdrop
+                                    src={
+                                        details.backdrop_path ?
+                                            `${secureBaseUrl}${details.backdrop_path}` :
+                                            ""//`${noPerson}`
```
Comment (kepkaklaudia) [inline src/features/movieDetails/index.js:43]: No need to write this comment

File: src/features/movieDetails/index.js [hunk 3]
```diff
@@ -1,77 +1,100 @@
...
+                            posterPath={`${secureBaseUrl}${details.poster_path}`}
+                            title={details.original_title}
+                            year={(details.release_date || "").substring(0, 4)}
+                            genres={
+                                (details.genres).map(genre => genre.name)
+                            }
+                            voteAverage={details.vote_average}
+                            voteCount={details.vote_count}
+                            description={details.overview}
+                            firstValue={"China USA"}
```
Comment (kepkaklaudia) [inline src/features/movieDetails/index.js:65]: `firstValue` should be `details.production_countries`
so in the code: 
`firstValue={(details.production_countries).map(country =>"${country.name}, ")}` ( change double quotes to ``)

**Review by kepkaklaudia — CHANGES_REQUESTED** [review]
Please correct the places indicated 😊
Good job! 😄

File: src/features/movieDetails/index.js (hunk 1 above)
Comment (q270311) [reply src/features/movieDetails/index.js:53]: aha, ok, You're right. It will be easy to change

File: src/features/movieDetails/index.js (hunk 2 above)
Comment (q270311) [reply src/features/movieDetails/index.js:43]: I deleted it

File: src/features/movieDetails/index.js (hunk 3 above)
Comment (q270311) [reply src/features/movieDetails/index.js:65]: Sorry, I forgot to come back to this

**Comment by q270311** [comment]
Thank you @kepkaklaudia  so much for review :)

**Review by kepkaklaudia — APPROVED** [review]

**Pull request merged**